### PR TITLE
adapt review comments; log pull request numbers; update core version to pick up RestClient fix

### PR DESCRIPTION
**Pull request by benj58xu**
adapt review comments for the previous pull request;
also log pull request numbers

### CODE REVIEW

File: src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java [hunk 1]
```diff
@@ -169,10 +169,11 @@ public void collect(Collector collector) {
         LOG.info("GitHubCollectorTask:collect start, total enabledRepos=" + enabledRepos.size());
         for (GitHubRepo repo : enabledRepos) {
             repoCount++;
-            String repoUrl = repo.getRepoUrl() + "/tree/" + repo.getBranch();
+            String repoUrl = repo==null?"null":(repo.getRepoUrl() + "/tree/" + repo.getBranch());
```
Comment (rvema) [inline src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java]: Can we use StringUtils.isEmpty in place of the null check?

File: src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java [hunk 2]
```diff
@@ -169,10 +169,11 @@ public void collect(Collector collector) {
...
             String statusString = "UNKNOWN";
-            long lastUpdated = repo.getLastUpdated();
+            long lastUpdated = repo==null?0:repo.getLastUpdated();
             try {
+                if (repo==null) throw new HygieiaException("Repository returned from github is null", HygieiaException.BAD_DATA);
```
Comment (rvema) [inline src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java]: shouldn't the null check be the first line inside the for loop? Also if the repo is null we need to log an error and continue to the next repo.

Comment (benj58xu) [reply src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java]: I would like to have the same log line in "finally" to log for every case, in order for splunk to report all data as well as statistics, that is why I do not use a different log line.

File: src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java (hunk 1 above)
Comment (benj58xu) [reply src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java]: See the response below

**Pull request merged**